swift/llm/sft.py

```diff
@@ -79,6 +90,27 @@ def llm_sft(args: SftArguments) -> Dict[str, Union[str, Any]]:
     logger.info(model_info)
     logger.info(model)
 
+    if use_torchacc():
+        import torchacc as ta
+
+        def get_ta_config():
```

Comment:
Please don't write this function in the main, may be utils

Reply:
move it to accelerator.py